### PR TITLE
2017 Events: Hip Hop Hacks & MusED

### PR DESCRIPTION
**Pull request by khoeger**
- Update Hip Hop Hacks 2017 .md file
- Initiate MusED 2017 .md file

### CODE REVIEW

**Comment by khoeger** [comment]
**Note** Adding proper formatting ("") to the event descriptors (title, date, description, address, etc.) fixed the past events formatting problem that we noticed, where Hip Hop Hacks did not migrate into past events.

**Comment by trevorknight** [comment]
> *Note* Adding proper formatting ("") to the event descriptors (title,
> date, description, address, etc.) fixed the past events formatting problem
> that we noticed, where Hip Hop Hacks did not migrate into past events.

Really? I didn't realize there was a problem?

File: _events/2017-08-05.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+description: "Description Coming shortly"
+date: 2017-08-05 12:00pm
+address: "45 W 18th St, New York, NY 10011"
+#rsvp: "http://eventbrite.link.or.similar"
+#image: "place-this-image-file-in-assets-slash-events.jpg"
+---
+
+# Schedule
+
+- 11:30 Doors open
```
Comment (trevorknight) [inline _events/2017-08-05.md:12]: In the spirit of the website always being in a state of presentable, how about "Announced soon" or something under Talks and Workshops?

Comment (khoeger) [reply _events/2017-08-05.md:12]: Ok! I'll go edit that.

**Comment by khoeger** [comment]
It didn't show up for me. It showed a title-less event that  could not be accessed, and had another date.

**Comment by trevorknight** [comment]
http://mmhpreview.com <- Looks good right now, no?

**Comment by trevorknight** [comment]
There could be a difference between your local environment and the Github
pages environment that could result in different behavior.

**Review by trevorknight — APPROVED** [review]

**Pull request merged**